config/root-config.in

```diff
@@ -558,7 +558,7 @@ while test $# -gt 0; do
     --git-revision)
       ### Output the git revision. Give up if gitinfo.txt can not be found.
       if test -r "${etcdir}/gitinfo.txt"; then
-         out="$out `head -m2 \"${etcdir}/gitinfo.txt\" | tail -n`"
+         out="$out `head -n 2 \"${etcdir}/gitinfo.txt\" | tail -n 1`"
```

Comment:
I think the space between n and the numbers is not needed, but it does not harm either.

Reply:
Other than that, I think there is still a problem. The etc/gitinfo.txt file for Ubuntu 22.04 binary release says:

```
tags/v6-30-02

Nov 27 2023, 19:50:38
```

so `head -n2 etc/gitinfo.txt | tail -n1`

returns an empty string.

Reply:
So it clearly does not match with the content of etc/gitinfo.txt that @bellenot showed in his post for Windows.